swap/peer.go

```diff
@@ -221,3 +230,19 @@ func (p *Peer) sendCheque() error {
 		Cheque: cheque,
 	})
 }
+
+// getBouncedCheque retrieves if a bounced cheque exists at address from the blockchain
```

Comment:
this comment should be very similar to the `Bounced` function, only that its receiver is a peer, not a contract. having a different comment could be confusing, let's have them aligned